mlflow/java/client/pom.xml

```diff
@@ -50,11 +50,6 @@
       <groupId>org.ini4j</groupId>
       <artifactId>ini4j</artifactId>
     </dependency>
-    <dependency>
-      <groupId>com.google.code.findbugs</groupId>
-      <artifactId>jsr305</artifactId>
```

Comment:
This library needs to be statically compiled into jar to get `@Nonnull` to work.